printing-services/backend/.env.example

```
@@ -0,0 +1,43 @@
+STRIPE_PUBLISHABLE_KEY="pk_test_your_stripe_publishable_key"
+STRIPE_WEBHOOK_SECRET="whsec_your_webhook_secret"
+
+# AWS S3 File Storage
+AWS_ACCESS_KEY_ID="your-aws-access-key"
+AWS_SECRET_ACCESS_KEY="your-aws-secret-key"
+AWS_REGION="us-east-1"
+AWS_S3_BUCKET="printing-services-uploads"
+
+# Blockchain Configuration
+POLYGON_RPC_URL="https://polygon-rpc.com"
+PRIVATE_KEY="your-wallet-private-key-for-contract-deployment"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)